# AUTHOR: Neset Unver Akmandor (NUA)
# E-MAIL: akmandor.n@northeastern.edu

world_frame_name: "world"
gz_model_msg_name: "/gazebo/model_states"

name_pkgs_ign: ["conveyor_belt","red_cube","green_cube","blue_cube", "actor"] 
name_pkgs_man: ["normal_pkg","long_pkg","longwide_pkg"] 

scan_data_path_pkgs_ign: ["models/point_cloud/conveyor_belt.json",
                          "models/point_cloud/red_cube.json",
                          "models/point_cloud/green_cube.json",
                          "models/point_cloud/blue_cube.json",
                          "models/point_cloud/actor.json"]

scan_data_path_pkgs_man: ["models/point_cloud/normal_pkg.json",
                          "models/point_cloud/long_pkg.json",
                          "models/point_cloud/longwide_pkg.json"]

map_resolution: 0.1